### DISCUSSION TITLE
Activate editor ONLY on Enter key down

### DISCUSSION THREAD

**Discussion by iamrakhmatov** [Comcast/react-data-grid#3443] [Q&A]
Right now every single example enables you to activate edit mode on any key press. 
What I want is to activate edit mode only when you press Enter key on the focused cell.

Is this possible?

**Comment by amanmahajan7** [comment]
You should be able to use `onCellKeyDown` to control when to edit the cell

```
onCellKeyDown={(args, event) => {
  if (args.mode === 'SELECT' && event.key !== 'Enter') {
     event.preventGridDefault();
  }
}}
```

**Reply by veddishant** [reply]
@amanmahajan7 Its works but cell navigation goes stop from arrow keys up-down-left-right and also tab not work any more, that's allow necessary.
I am considering this data grid for making desktop web software its only allow to cell edit on Enter Key press to prevent by mistake type on any cell value change.  you have any solution on that please suggest here same.
